pyfar/classes/audio.py

```diff
@@ -108,6 +108,28 @@ def reshape(self, newshape):
 
         return reshaped
 
+    def transpose(self, *axes):
+        """Return a transposed copy of the audio object.
+
+        Parameters
+        ----------
+        axes : empty, iterable of ints, or n ints
+            leaving the argument empty will reverse the order of `self.caxes`.
+            iterable of ints: `i` in the `j`-th place of the interable means
+            that `self._data`'s `i`-th axis becomes the transposed datas `j`-th
```

Comment:
I would avoid mentioning `_data` in the docstring because we usually hide this from the user. Maybe soemthing like '... means that the `i`-ths axis of the time/frequency data becomes ...'